src/com/ka/spreadsheet/diff/CellValue.java

```diff
@@ -5,6 +5,7 @@
   private final boolean hasFormula;
   private final String formula;
   private final Object value;
+  private final ICellValueTransformer valueTransformer = new PropertySetupColCellValueTransformer();
```

Comment:
Just curious why instantiating this globally?

Reply:
Yeah you are right, not a big deal for what we are trying to do, but I will change this to a static method instead.